src/v/cluster/health_manager.cc

```diff
@@ -197,6 +197,11 @@ ss::future<> health_manager::do_tick() {
         if (ok) {
             ok = co_await ensure_topic_replication(model::transform_offsets_nt);
         }
+
+        if (ok) {
+            ok = co_await ensure_topic_replication(
+              model::kafka_audit_logging_nt);
+        }
```

Comment:
So `ensure_topic_replication` may up-replicate the topic regardless of the settings set by the user (e.g. user sets RF=1 for this and _target_replication_factor is 3).  So.... probably don't want to do this.